packages/grafana-ui/src/components/PageLayout/PageToolbar.tsx

```diff
@@ -56,7 +47,7 @@ export const PageToolbar: FC<Props> = React.memo(
       <div className={mainStyle}>
         {pageIcon && !onGoBack && (
           <div className={styles.pageIcon}>
-            <Icon name={pageIcon} size="lg" />
+            <Icon name={pageIcon} size="lg" aria-hidden />
```

Comment:
Maybe this should have an alt or a title that clarifies that this context has to do with dashboards instead. Otherwise it is not entirely clear. Visually it is what the icon does.

Reply:
Or maybe an aria-label that describes that this section is for searching dashboards.

Reply:
The toolbar itself already has an aria-label of "Dashboard navigation", so I think any additional labels to that effect might be annoying more than anything else for people using a screen reader. What do you think?

Reply:
Oh, didn't see that! That makes sense, yeah